modules/telemetry/telemetry.py

```diff
@@ -291,7 +298,7 @@ def play_mission(self, mission_name: str | None) -> None:
             raise ReplayPlaybackError
 
         mission_file = mission_path(mission_name, self.missions_dir)
-        if mission_name is not None and mission_file not in self.status.replay.mission_files_list:
```

Comment:
This is used to catch the front end from sending a blank mission name and crashing the program.